bot/cogs/bot.py

```diff
@@ -60,6 +62,27 @@ def __init__(self, bot: AutoShardedBot):
 
         await ctx.invoke(self.info)
 
+    async def on_message(self, msg: Message):
+        if msg.content.count("\n") >= 3:  # more than three lines
+            try:
+                tree = ast.parse(msg.content)
+                # Attempts to parse the message into an AST node.
+                # Invalid Python code will raise a SyntaxError.
+                if not all(isinstance(node, ast.Expr) for node in tree.body):
+                    # we dont want multiple lines of single words like:
+                    '''
```

Comment:
Is a docstring really necessary here?

Reply:
How else would one do multiline comments without 
# 
# 
# 
or should i use that?

Reply:
You don't really need an example. I think just explaining what the check does should suffice.

Reply:
yeah don't use docstrings like this. if you did need a comment of that many lines, you should have used multiple # comments, but I agree with Ap that you don't really need that in this case. Just say it with words.